Content.Client/UserInterface/Systems/Character/CharacterUIController.cs

```diff
@@ -126,8 +149,7 @@ private void CharacterUpdated(CharacterData data)
                 conditionControl.ProgressTexture.Texture = condition.SpriteSpecifier.Frame0();
                 conditionControl.ProgressTexture.Progress = condition.Progress;
 
-                conditionControl.Title.Text = condition.Title;
-                conditionControl.Description.Text = condition.Description;
+                conditionControl.Title.Text = WordWrap(condition.Title + " " + condition.Description, MaxObjectiveLineLength) + "\n";
```

Comment:
The condition.Title and condition.Description shouldn't be joined like this - in the PR screenshots, the text "Die a glorious death Die" is pretty confusing, for example. Keep the old code's style of using two different UI elements so that they remain separated and can apply different styles, etc..

Also, don't manually add newlines to text for padding. Add a margin or padding or a spacer to the control instead.